src/index.js

```diff
@@ -4,7 +4,7 @@ import './index.css';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
 import { AppServicesProvider } from './services/pure-di';
-import { HashRouter as Router } from 'react-router-dom';
+import { BrowserRouter as Router } from 'react-router-dom';
```

Comment:
😂

Reply:
Nice! it was a simple change.

Did you validate if `#` based URLs do not work auto-magically?

Reply:
Yes, but sadly no, they all get redirected to `/`